charts/pythie/templates/pythie-serving-deployment.yaml

```diff
@@ -42,6 +42,9 @@ spec:
       {{- if $.Values.priorityClassName }}
       priorityClassName: {{ $.Values.priorityClassName }}
       {{- end }}
+      {{- if $.Values.pythieServing.imagePullSecrets }}
+      imagePullSecrets: {{ $.Values.pythieServing.imagePullSecrets }}
+      {{- end }}
```

Comment:
hello @desaintmartin !

I'd like to be able to pull a pythie-serving image from a private repo (because I'd need to create a custom image with wiremind-only logic that I don't want to publish to the official pythie-serving repo).

I can already change the repo L68 and I would have put the pullSecrets there too (eg L69) but that doesn't seem to work so I put here, doesn't that make sense or is there a better solution?